lambda/variables.tf

```diff
@@ -0,0 +1,90 @@
+variable "function_name" {
+  description = "The name of the lambda function"
+}
+
+variable "handler" {
+  description = "The lambda function handler"
+}
+
+variable "kms_key_arn" {
+  description = "The kms key to use to encrypt environment variables if necessary"
+  default     = ""
+}
+
+variable "encrypted_env_vars" {
+  default     = {}
+  type        = map(string)
+  description = "A map of environment variables in the form ENV_VAR_NAME -> plaintext value These will be encrypted"
+}
+
+variable "plaintext_env_vars" {
+  default     = {}
+  type        = map(string)
+  description = "A map of environment variables in the form ENV_VAR_NAME -> plaintext value These will not be encrypted"
+}
+
+variable "runtime" {
+  description = "The lambda runtime, for example java11"
+}
+
+variable "timeout_seconds" {
+  default = 15
+}
+
+variable "memory_size" {
+  default = 1024
+}
```

Comment:
Why so big?

Reply:
It was copied from TDR and they have lambdas that need a lot of memory. I've set it to 256 because any less than that and the Java lambdas won't run.